.github/workflows/create-replacement-pr.yml

```diff
@@ -150,29 +152,47 @@ jobs:
         with:
           signing_key: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY }}
 
-      - name: Pull PR${{ inputs.upload && ' and upload bottles to GitHub Packages' || '' }}
+      - name: Pull PR
+        id: pr-pull
         working-directory: ${{ steps.set-up-homebrew.outputs.repository-path }}
         env:
-          BREWTESTBOT_NAME_EMAIL: "${{ steps.git-user-config.outputs.name }} <${{ steps.git-user-config.outputs.email }}>"
-          HOMEBREW_GPG_PASSPHRASE: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY_PASSPHRASE }}
```

Comment:
Thinking about this some more, I suspect we need this (and `--committer`) if `autosquash` is enabled. Thoughts, @Bo98?

Reply:
yes